paddle/fluid/framework/operator.cc

```diff
@@ -155,7 +163,20 @@ RuntimeContext::RuntimeContext(const VariableNameMap& innames,
   }
 }
 
+void OperatorBase::PreHook(const Scope& scope, const platform::Place& place) {
+  auto attrName = OpProtoAndCheckerMaker::OpCreationCallstackAttrName();
+  if (HasAttr(attrName)) {
+    auto& callstack = Attr<std::vector<std::string>>(attrName);
+    platform::PythonDebugSupport::GetInstance()->SetInformation(callstack);
```

Comment:
This python stack should be thrown all the time when there is an exception

Reply:
ok